elastic-search-support/src/test/resources/application.yml

```diff
@@ -1,72 +0,0 @@
-elasticsearch:
```

Comment:
how come this change?

Reply:
This file was outdated, hence removed the mapping and renamed the file to application-default.yml. Now the mapping for tests will be picked up from main/resource/application.yml.